custom-connectors/Actsoft/apiDefinition.swagger.json

```diff
@@ -0,0 +1,14408 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "AT&T Workforce Manager",
+    "description": "AT&T Workforce Manager(WFM) is an all-in-one service management platform that helps you more effectively manage your remote teams of employees and general business operations away from the office. As a cloud-based solution, WFM enables remote data collection and displays it via its web app dashboard, providing managers with the additional insight they need to create smarter strategies for increased efficiency, productivity, and reduced costs while coordinating their mobile workforces.",
+    "contact": {
+      "name": "Actsoft Support",
+      "url": "https://www.actsoft.com/self-help-wizards",
+      "email": "support@actsoft.com"
+    }
+  },
+  "host": "qacabdevapi.actsoft.com",
```

Comment:
@rdameron We do not certify a connector which has a dev host URL. Please update it to Production host URL